test/runAnalysis/runAnalysis.py

```diff
@@ -310,8 +325,12 @@ def main():
 					from samples import backgrounds_muons_2018 as samples
 				else:	
 					from samples import backgrounds_muons_2017 as samples 
+			if args.add2016:
+				from samples import add_2016 as samples
 			lumi_mask = ""
 			GT = "94X_mc2017_realistic_v14"
+			if args.add2016:
+				GT = "84X_mcRun2_asymptotic_2016_TranchelV_v6"
```

Comment:
As far as I am aware our processing of the miniAOD files does nothing were the information contained in the global tags is actually used, so this doesn't really matter. But we should in principle keep in mind that we should add here the correct GT configuration for all the years of data taking. Also, I'm pretty sure there is no CMSSW 84X, so I have no idea what kind of tag you picked here ;)
But in general, could you have a look at this table and add all the GTs for data and MC analysis here https://twiki.cern.ch/twiki/bin/viewauth/CMS/PdmVAnalysisSummaryTable
If there are two, please pick the second one.

Reply:
Thanks a lot! I have just fixed this!